5/jamie-white-captions.srt

```diff
@@ -220,58 +221,59 @@ there's quite a few of them.
 
 52
 00:04:04,260 --> 00:04:08,525
-Because I'm using a local copy of Ember CLI inaudible.
+Because I'm using a local copy of Ember CLI–
 
 53
 00:04:08,925 --> 00:04:10,137
-I got it off Github.
+I cloned it off Github...
 
 54
 00:04:11,300 --> 00:04:13,460
-With npm, what you tend to do
+With NPM, what you tend to do
```

Comment:
I don't think I've seen NPM capitalized like that before. I thought it was always all lowercase. Even the company "npm, Inc." uses lowercase, so I think that's probably the way to go.